packages/TorneloScoresheet/src/hooks/appMode/tablePairingState.ts

```diff
@@ -11,6 +11,8 @@ type TablePairingStateHookType = [
   TablePairingMode,
   {
     goToRecording: (currentPlayer: PlayerColour) => void;
+
+    goToArbiterGameMode: () => void;
```

Comment:
Same here we need to pass through the state's data or it will be lost forever :(